### ISSUE TITLE
User story - Unique profile URL

### ISSUE THREAD

**Issue by stephendawsondev** [stephendawsondev/StackPortfolio#16]
### If applicable, add a related Epic.

Epic: #14 

### User Story

As a **user**, I can **create a unique profile URL** so that I can **easily share my profile with others**.

### Acceptance Criteria

- The unique URL should be the user's username, so it should follow the structure `https://www.stackportfolio.io/user/<username>`

### Please ensure you've completed these tasks before marking the issue as done.

- [x] Reviewed the user story for clarity and completeness.
- [x] Identified and linked related issues or epics.
- [x] Created or updated the necessary documentation.
- [x] Written unit and/or integration tests for functionality.
- [x] All code changes have been manually tested to ensure they meet the acceptance criteria.
- [x] Updated the issue with any relevant notes or feedback (to date).
- [x] Merged into the main branch and deployed to production (if applicable).

### Please add any further tasks that need to be completed before the issue can be marked as done.

- [x] Ensure that the slug used is the user's ID and it's unique

**Comment by stephendawsondev** [comment]
This has now been added and users' profiles can be viewed by navigating to `/user/<slug>`.
